projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.spec.ts

```diff
@@ -78,7 +91,9 @@ describe('ListViewComponent', () => {
         TestBed.configureTestingModule({
             declarations: [
                 ListViewComponent,
-                TestParentComponent
+                TestParentComponent,
```

Comment:
It worked without. Should I add it?

Reply:
I saw, that I added the `dsp-progress-indicator` mock to the wrong files. I'll fix it

Reply:
Fixed in 9f58652

Reply:
I think it really depends on when `dsp-progress-indicator` is needed. Since all async HTTP calls are mocked using `Observables` that are are actually [synchronous](https://angular.io/guide/testing-components-scenarios#testing-with-a-spy), it may not be needed.

Reply:
also if there are conditions in the template that load more components, it really depends on how the test looks. (interactions that change these conditions)

Reply:
I understand. May I merge it now?

Reply:
yes, I have already approved the PR :-)